### DISCUSSION TITLE
Add Possibility to add Outside Collaborator

### DISCUSSION THREAD

**Discussion by Felix83000** [community/community#10806] [Projects and Issues] [answered]
Hello,

It will be great to have the possibility to add Outside Collaborators to the project. 

For now, it is restricted to inside member of an Organization.

Regards,
Félix

**Comment by azenMatt** [comment, marked as answer]
Hi - this is now possible 😄 

you just have to make sure the user you would like to add has outside collaborator access to at least one repo in your org
